woff1/tools/validator/validator.py

```diff
@@ -2725,17 +2725,18 @@ def main():
     options.outputFormat = "html"
     options.testGroups = None # don't expose this to the commandline. it's for testing only.
     if outputDirectory is not None and not os.path.exists(outputDirectory):
-        print "Directory does not exist:", outputDirectory
+        print("Directory does not exist:", outputDirectory)
         sys.exit()
     for fontPath in args:
         if not os.path.exists(fontPath):
-            print "File does not exist:", fontPath
+            print("File does not exist:", fontPath)
             sys.exit()
         else:
-            print "Testing: %s..." % fontPath
+            print("Testing: %s..." % fontPath)
             fontPath = fontPath.decode("utf-8")
             outputPath, report = validateFont(fontPath, options)
-            print "Wrote report to: %s" % outputPath
+            print("Wrote report to: %s" % outputPath)
```

Comment:
Uses the `print_function` from `__future__` in Python 2 in order to provide compatibility with the Python 3 `print` function.